cypress/plugins/index.js

```diff
@@ -1,7 +1,20 @@
 const plugins = require('@dhis2/cli-utils-cypress/plugins')
 
 module.exports = (on, config) => {
```

Comment:
Interesting, so this is how we can solve the new cookie policy?

Reply:
yes, I'll try to put that in to the cypress repo and replace it here